.gitignore

```diff
@@ -26,3 +26,4 @@
 /configure
 /gen/
 /Makefile
+*swp
```

Comment:
This conflicts with master.

I would recommend to add this to your personal global gitignore instead (see e.g. https://stackoverflow.com/a/5724484/928031 for how to create one), as not everybody is using your editor, while you are probably using it everywhere; so it's better to exclude it for every repo you work with?